queue.go

```diff
@@ -140,6 +152,30 @@ func NewOrOpen(name string, dirPath string, itemsPerSegment int, builder func()
 	return New(name, dirPath, itemsPerSegment, builder)
 }
 
+// Close releases the lock on the queue directory rendering it unusable for further usage.
```

Comment:
Can you clarify this comment a bit?    Maybe just extend the sentence with "by this instance".

Reply:
Done.